src/Tickets/Emails/Email_Handler.php

```diff
@@ -197,4 +197,22 @@ public function add_settings_per_email( array $fields ): array {
 		return $fields;
 	}
 
+	/**
+	 * Get email by ID.
+	 * 
+	 * @since TBD
+	 * 
+	 * @param string $id ID of email.
+	 * 
+	 * @return Email_Abstract|boolean Email object or false if it does not exist.
+	 */
+	public function get_email_by_id( $id ) {
+		$emails = $this->get_all();
```

Comment:
We can maybe use `array_search`?

Reply:
`$emails` is an array of objects in this case. Can `array_search` be used in this case?